File: app/eclipse/futurehome/src/no/futurehome/futurehome_app/widget/helpers/JsonOptsKt.java
package no.futurehome.futurehome_app.widget.helpers;

import kotlin.Metadata;
import kotlin.jvm.internal.Intrinsics;
import org.json.JSONArray;
import org.json.JSONObject;

@Metadata(d1 = {"\0000\n\000\n\002\020\016\n\002\b\002\n\002\030\002\n\002\b\002\n\002\020\t\n\002\b\003\n\002\030\002\n\000\n\002\020\b\n\002\b\002\n\002\020\013\n\002\b\002\032\030\020\002\032\004\030\0010\0012\006\020\003\032\0020\0042\006\020\005\032\0020\001\032\035\020\006\032\004\030\0010\0072\006\020\003\032\0020\0042\006\020\005\032\0020\001¢\006\002\020\b\032\030\020\t\032\004\030\0010\0042\006\020\003\032\0020\0042\006\020\005\032\0020\001\032\030\020\n\032\004\030\0010\0132\006\020\003\032\0020\0042\006\020\005\032\0020\001\032\035\020\f\032\004\030\0010\r2\006\020\003\032\0020\0042\006\020\005\032\0020\001¢\006\002\020\016\032\035\020\017\032\004\030\0010\0202\006\020\003\032\0020\0042\006\020\005\032\0020\001¢\006\002\020\021\"\016\020\000\032\0020\001XT¢\006\002\n\000¨\006\022"}, d2 = {"TAG", "", "optStringFromJson", "json", "Lorg/json/JSONObject;", "key", "optLongFromJson", "", "(Lorg/json/JSONObject;Ljava/lang/String;)Ljava/lang/Long;", "optJsonFromJson", "optJsonArrayFromJson", "Lorg/json/JSONArray;", "optIntFromJson", "", "(Lorg/json/JSONObject;Ljava/lang/String;)Ljava/lang/Integer;", "optBooleanFromJson", "", "(Lorg/json/JSONObject;Ljava/lang/String;)Ljava/lang/Boolean;", "app_release"}, k = 2, mv = {2, 1, 0}, xi = 48)
public final class JsonOptsKt {
  private static final String TAG = "Opts";
  
  public static final Boolean optBooleanFromJson(JSONObject paramJSONObject, String paramString) {
    Intrinsics.checkNotNullParameter(paramJSONObject, "json");
    Intrinsics.checkNotNullParameter(paramString, "key");
    boolean bool = paramJSONObject.isNull(paramString);
    Exception exception2 = null;
    if (bool)
      return null; 
    try {
      Boolean bool1 = Boolean.valueOf(paramJSONObject.getBoolean(paramString));
    } catch (Exception exception1) {
      Logger.Companion companion = Logger.Companion;
      StringBuilder stringBuilder = new StringBuilder("optBooleanFromJson, exception: ");
      stringBuilder.append(exception1);
      companion.e("Opts", stringBuilder.toString());
      exception1 = exception2;
    } 
    return (Boolean)exception1;
  }
  
  public static final Integer optIntFromJson(JSONObject paramJSONObject, String paramString) {
    StringBuilder stringBuilder1;
    Intrinsics.checkNotNullParameter(paramJSONObject, "json");
    Intrinsics.checkNotNullParameter(paramString, "key");
    boolean bool = paramJSONObject.isNull(paramString);
    StringBuilder stringBuilder2 = null;
    if (bool)
      return null; 
    try {
      Integer integer = Integer.valueOf(paramJSONObject.getInt(paramString));
    } catch (Exception exception) {
      Logger.Companion companion = Logger.Companion;
      stringBuilder1 = new StringBuilder("optIntFromJson, exception: ");
      stringBuilder1.append(exception);
      companion.e("Opts", stringBuilder1.toString());
      stringBuilder1 = stringBuilder2;
    } 
    return (Integer)stringBuilder1;
  }
  
  public static final JSONArray optJsonArrayFromJson(JSONObject paramJSONObject, String paramString) {
    StringBuilder stringBuilder1;
    Intrinsics.checkNotNullParameter(paramJSONObject, "json");
    Intrinsics.checkNotNullParameter(paramString, "key");
    boolean bool = paramJSONObject.isNull(paramString);
    StringBuilder stringBuilder2 = null;
    if (bool)
      return null; 
    try {
      JSONArray jSONArray = paramJSONObject.getJSONArray(paramString);
    } catch (Exception exception) {
      Logger.Companion companion = Logger.Companion;
      stringBuilder1 = new StringBuilder("optJsonArrayFromJson, exception: ");
      stringBuilder1.append(exception);
      companion.e("Opts", stringBuilder1.toString());
      stringBuilder1 = stringBuilder2;
    } 
    return (JSONArray)stringBuilder1;
  }
  
  public static final JSONObject optJsonFromJson(JSONObject paramJSONObject, String paramString) {
    Intrinsics.checkNotNullParameter(paramJSONObject, "json");
    Intrinsics.checkNotNullParameter(paramString, "key");
    boolean bool = paramJSONObject.isNull(paramString);
    Exception exception2 = null;
    if (bool)
      return null; 
    try {
      paramJSONObject = paramJSONObject.getJSONObject(paramString);
    } catch (Exception exception1) {
      Logger.Companion companion = Logger.Companion;
      StringBuilder stringBuilder = new StringBuilder("optJsonFromJson, exception: ");
      stringBuilder.append(exception1);
      companion.e("Opts", stringBuilder.toString());
      exception1 = exception2;
    } 
    return (JSONObject)exception1;
  }
  
  public static final Long optLongFromJson(JSONObject paramJSONObject, String paramString) {
    Intrinsics.checkNotNullParameter(paramJSONObject, "json");
    Intrinsics.checkNotNullParameter(paramString, "key");
    boolean bool = paramJSONObject.isNull(paramString);
    Exception exception2 = null;
    if (bool)
      return null; 
    try {
      Long long_ = Long.valueOf(paramJSONObject.getLong(paramString));
    } catch (Exception exception1) {
      Logger.Companion companion = Logger.Companion;
      StringBuilder stringBuilder = new StringBuilder("optLongFromJson, exception: ");
      stringBuilder.append(exception1);
      companion.e("Opts", stringBuilder.toString());
      exception1 = exception2;
    } 
    return (Long)exception1;
  }
  
  public static final String optStringFromJson(JSONObject paramJSONObject, String paramString) {
    Logger.Companion companion1;
    Intrinsics.checkNotNullParameter(paramJSONObject, "json");
    Intrinsics.checkNotNullParameter(paramString, "key");
    boolean bool = paramJSONObject.isNull(paramString);
    Logger.Companion companion2 = null;
    if (bool)
      return null; 
    try {
      String str = paramJSONObject.getString(paramString);
    } catch (Exception exception) {
      companion1 = Logger.Companion;
      StringBuilder stringBuilder = new StringBuilder("optStringFromJson, exception: ");
      stringBuilder.append(exception);
      companion1.e("Opts", stringBuilder.toString());
      companion1 = companion2;
    } 
    return (String)companion1;
  }
}

File: app/futurehome-kotlin/no/futurehome/futurehome_app/widget/helpers/JsonOptsKt.kt
package no.futurehome.futurehome_app.widget.helpers

import org.json.JSONArray
import org.json.JSONObject

private const val TAG: String = "Opts"

public fun optBooleanFromJson(json: JSONObject, key: String): Boolean? {
   if (var0.isNull(var1)) {
      return null;
   } else {
      try {
         var6 = var0.getBoolean(var1);
      } catch (var5: Exception) {
         val var7: Logger.Companion = Logger.Companion;
         val var4: StringBuilder = new StringBuilder("optBooleanFromJson, exception: ");
         var4.append(var5);
         var7.e("Opts", var4.toString());
         var6 = null;
      }

      return var6;
   }
}

public fun optIntFromJson(json: JSONObject, key: String): Int? {
   if (var0.isNull(var1)) {
      return null;
   } else {
      try {
         var7 = var0.getInt(var1);
      } catch (var5: Exception) {
         val var8: Logger.Companion = Logger.Companion;
         val var6: StringBuilder = new StringBuilder("optIntFromJson, exception: ");
         var6.append(var5);
         var8.e("Opts", var6.toString());
         var7 = null;
      }

      return var7;
   }
}

public fun optJsonArrayFromJson(json: JSONObject, key: String): JSONArray? {
   if (var0.isNull(var1)) {
      return null;
   } else {
      try {
         var7 = var0.getJSONArray(var1);
      } catch (var5: Exception) {
         val var4: Logger.Companion = Logger.Companion;
         val var6: StringBuilder = new StringBuilder("optJsonArrayFromJson, exception: ");
         var6.append(var5);
         var4.e("Opts", var6.toString());
         var7 = null;
      }

      return var7;
   }
}

public fun optJsonFromJson(json: JSONObject, key: String): JSONObject? {
   if (var0.isNull(var1)) {
      return null;
   } else {
      try {
         var0 = var0.getJSONObject(var1);
      } catch (var5: Exception) {
         val var7: Logger.Companion = Logger.Companion;
         val var4: StringBuilder = new StringBuilder("optJsonFromJson, exception: ");
         var4.append(var5);
         var7.e("Opts", var4.toString());
         var0 = null;
      }

      return var0;
   }
}

public fun optLongFromJson(json: JSONObject, key: String): Long? {
   if (var0.isNull(var1)) {
      return null;
   } else {
      try {
         var6 = var0.getLong(var1);
      } catch (var5: Exception) {
         val var7: Logger.Companion = Logger.Companion;
         val var4: StringBuilder = new StringBuilder("optLongFromJson, exception: ");
         var4.append(var5);
         var7.e("Opts", var4.toString());
         var6 = null;
      }

      return var6;
   }
}

public fun optStringFromJson(json: JSONObject, key: String): String? {
   if (var0.isNull(var1)) {
      return null;
   } else {
      try {
         var7 = var0.getString(var1);
      } catch (var5: Exception) {
         val var6: Logger.Companion = Logger.Companion;
         val var4: StringBuilder = new StringBuilder("optStringFromJson, exception: ");
         var4.append(var5);
         var6.e("Opts", var4.toString());
         var7 = null;
      }

      return var7;
   }
}
